tests/e2e/test_rosacli_ingress.go

```diff
@@ -508,3 +508,63 @@ var _ = Describe("Edit ingress",
 
 			})
 	})
+var _ = Describe("Delete ingress validations",
+	labels.Feature.Ingress,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			clusterID      string
+			rosaClient     *rosacli.Client
+			ingressService rosacli.IngressService
+			isHosted       bool
+		)
+
+		BeforeEach(func() {
+			By("Get the cluster")
+			clusterID = config.GetClusterID()
+			Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			ingressService = rosaClient.Ingress
+
+			By("Check cluster is hosted")
+			var err error
+			isHosted, err = rosaClient.Cluster.IsHostedCPCluster(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("will fail as expected on incorrect deletions (negative) - [id:38780]",
+			labels.Medium, labels.Runtime.Day2,
+			func() {
+				if isHosted {
+					SkipNotClassic()
+				}
+				By("Fail to delete ingress when no ingress or cluster are passed")
+				output, err := ingressService.DeleteIngress("", "")
+				Expect(err).To(HaveOccurred())
+				Expect(output.String()).Should(
+					ContainSubstring("Expected exactly one command line parameter containing the id of the ingress"))
```

Comment:
Why is that not `required flag(s) \"cluster\" not set` ?

Reply:
Hmm I guess I don't know which one I think makes more sense to prioritize in this case. Do we generally show that error with top priority? If so I will cut a ticket for the developers. In the mean time I've added another case to confirm that the `"cluster" not set` error does occur when an Ingress is passed but not a cluster